services.yaml

```diff
@@ -12,3 +12,5 @@
 - QuickSight
 - EC2
 - S3
+- AWS SDK for .NET
+- AWS Cloud​Formation
```

Comment:
Please shorten to just CloudFormation to match other services.

Reply:
@jflasher done!!!